charts/cloudflared/Chart.yaml

```diff
@@ -3,8 +3,8 @@ maintainers:
   - name: kubitodev
     url: https://kubito.dev
 apiVersion: v2
-appVersion: 2025.5.0
-version: 1.6.3
+appVersion: 2025.2.1
+version: 1.7.0
```

Comment:
Please fetch latest main changes before we merge this so we don't revert to an earlier version here.

Reply:
You mean for the `appVersion`?

Reply:
Yeah, merge the upstream main branch into yours please.